client/src/App.jsx

```diff
@@ -16,7 +16,33 @@ const theme = createMuiTheme({
     secondary: {
       main: '#F15852',
     },
-    // error: will use the default color
+    active: {
+      backgroundColor: '#69BE28',
+    },
+    // to use active use style={theme.palette.active}
+  },
+  typography: {
```

Comment:
Really big object with lots of lines. Might be worth decoupling a bit more to make the theme reusable.

Reply:
good shout, have moved theme into a new file

Reply:
added a test too